src/Components/Artwork/Details/Details.tsx

```diff
@@ -605,8 +605,6 @@ export const DetailsFragmentContainer = createFragmentContainer(Details, {
       }
       ...PrimaryLabelLine_artwork
       ...BidTimerLine_artwork
-      ...SaveButton_artwork
```

Comment:
This is big - removing these fragments from the standard `Artwork` component query (used in rails + grids).